### DISCUSSION TITLE
Visualizers state

### DISCUSSION THREAD

**Discussion by MerlinTheProgramist** [clangen/musikcube#693] [General]
It's been bothering me since I started using musikcube, there is a shortcut for a visualizer, but no other documentation about this feature. What is the current development state of it and what are the plans?

**Comment by xXMariahScaryXx** [comment]
bump, been trying to figure out how the hell to get one working but it seemingly just doesn't exist

**Comment by 237dmitry** [comment]
Visualization is available only for Windows.
